### PR TITLE
refactor: postinstall script

### PR DESCRIPTION
**Pull request by vchimev**
chore: update package.json

### CODE REVIEW

File: postinstall.js [hunk 1]
```diff
@@ -1,26 +1,135 @@
...
+    let spawnArgs = [];
+    let command = "";
+    if (isWin) {
+        command = "cmd.exe"
+        spawnArgs = ["/c", "npm", "install"];
+    } else {
+        command = "npm"
+        spawnArgs = ["install"];
+    }
+    const npm = childProcess.spawnSync(command, spawnArgs, { cwd: cwd, stdio: "inherit" });
```
Comment (sis0k0) [inline postinstall.js]: You don't need to assign that to a variable.
Also `cwd: cwd` can be shortened to just `cwd`

File: postinstall.js [hunk 2]
```diff
@@ -1,26 +1,135 @@
...
+}
+
+var pendingNpmInstall = false;
```
Comment (sis0k0) [inline postinstall.js]: let

File: postinstall.js [hunk 3]
```diff
@@ -1,26 +1,135 @@
...
+
+function configureDevDependencies(packageJson, adderCallback) {
+    if (!packageJson.devDependencies) {
+        packageJson.devDependencies = {};
+    }
+
+    let devDependencies = packageJson.devDependencies;
```
Comment (sis0k0) [inline postinstall.js]: const

File: postinstall.js [hunk 4]
```diff
@@ -1,26 +1,135 @@
...
+
+        const entries = new Array();
+        readdirSync(resolve(src)).forEach(entry => {
+            entries.push(entry);
+        });
+
+        entries.forEach(entry => {
+            const source = resolve(src,  entry);
+            const destination = resolve(dest, entry);
+            console.info("Copying " + source + " to " + destination + " ...");
```
Comment (sis0k0) [inline postinstall.js]: Should the user see that info by default?

**Comment by vchimev** [comment]
run ci

**Comment by SvetoslavTsenov** [comment]
As I can see the script is not going to execute on post-install when we develop the plugin which is great and exactly as I expected.  But in this case, do we need this to be written in pure javascript, since, at the end using typescript we can benefit from it?

**Pull request merged**